src/callbacks/sirius.js

```diff
@@ -17,93 +17,60 @@ import { SIRIUS_PROXY_PATH } from '../config'
 const emptyObject = (obj) => R.is(Object, obj) && R.pipe(R.keys, R.propEq('length', 0))
 
 /**
- * Fittable widget instance
- */
-var f
-
-/**
- * Main application parameter, specifying the view
- * Choices are: person, room, course
- * @type {string}
- */
-var view = 'person'
-
-/**
- * Parameter for the selected view. Used for specifying the room or course.
- * @type {string}
- */
-var parameter = ''
-
-/**
  * Logged user's username.
  */
-var username = ReactCookie.load('oauth_username')
+const username = ReactCookie.load('oauth_username')
 
 /**
  * URL of proxy for Sirius API
  * @type {string}
  */
-var siriusAPIUrl = `${getBaseUri()}${SIRIUS_PROXY_PATH}/`
-
-/**
- * Currently selected application language setting
- * @type {string}
- */
-var appLocale = 'cs'
-
-
-var viewPaths = {
-  course: 'courses',
-  person: 'people',
-  room: 'rooms',
-}
+const siriusAPIUrl = `${getBaseUri()}${SIRIUS_PROXY_PATH}/`
 
 // TODO: This should be removed!
-var defaultLimit = 200
+const defaultLimit = 200
 
-var STATUS_ERROR_TYPES = {
+const STATUS_ERROR_TYPES = {
   0: 'connection',
   401: 'unauthorized',
   403: 'access',
   404: 'notfound',
-  500: 'servererror'
+  500: 'servererror',
 }
+
 function generateError (status, message = 'No message specified') {
   const error = new Error(message)
   if (status in STATUS_ERROR_TYPES) {
     error.type = STATUS_ERROR_TYPES[status]
-    if (error.type === 'notfound' && view === 'person' && parameter === username) {
+    /* if (error.type === 'notfound' && view === 'person' && parameter === username) {
       error.type = 'own' + error.type
-    }
+    } */
```

Comment:
Because we cannot handle this, but there is an intent to distinguish between access error on own resource.